### PR TITLE
MODULES.md update

### PR DESCRIPTION
**Pull request by rafzei**

### CODE REVIEW

File: docs/home/howto/MODULES.md [hunk 1]
```diff
@@ -2,99 +2,76 @@
...
 ## Introduction
 
-In version 0.8 of epiphany we are introducing modules. Modularisation of epiphany environment will result in: 
- * smaller code bases for separate areas, 
- * simpler and faster test process, 
- * interchangeability of elements providing similar functionality (eg.: different kubernetes providers), 
- * faster and more focused release cycle. 
- 
-Those and multiple other factors (eg.: readability, reliability) influence this direction of changes. 
+In version 0.8 of epiphany we are introducing modules. Modularisation of epiphany environment will result in:
```
Comment (atsikham) [inline docs/home/howto/MODULES.md:5]:
```suggestion
In version 0.8 of epiphany we are introducing modules. Modularization of epiphany environment will result in:
```

**Review by atsikham — APPROVED** [review]
Content is ok for me, but syntax/grammar should be checked.

File: docs/home/howto/MODULES.md (hunk 1 above)
Comment (rafzei) [reply docs/home/howto/MODULES.md:5]: I think is the same. UK:s, US:z. But sure, I can change it.

Comment (rafzei) [reply docs/home/howto/MODULES.md:5]: I think it's the same. UK:s, US:z. But sure, I can change it.

**Review by atsikham — DISMISSED** [review]

File: docs/home/howto/MODULES.md [hunk 2]
```diff
@@ -2,99 +2,76 @@
...
 ## New scenarios
 
-In 0.8 we introduce possibility to use AKS or EKS as kubernetes providers. That is introduced with modules mechanism, so we introduced first four modules: 
- * [Azure Basic Infrastructure](https://github.com/epiphany-platform/m-azure-basic-infrastructure) (AzBI) module
- * [Azure AKS](https://github.com/epiphany-platform/m-azure-kubernetes-service) (AzKS) module
- * [AWS Basic Infrastructure](https://github.com/epiphany-platform/m-aws-basic-infrastructure) (AwsBI) module
- * [AWS EKS](https://github.com/epiphany-platform/m-aws-kubernetes-service) (AwsKS) module
- 
-Those 4 modules with the Classic Epiphany used with `any` provider allow change of on-prem kubernetes cluster with managed kubernetes services. 
+In 0.8 we introduce the possibility to use AKS or EKS as Kubernetes providers. That is introduced with modules mechanism, so we introduced the first four modules:
```
Comment (plirglo) [inline docs/home/howto/MODULES.md:32]: introduce synonyms, maybe like this?
In 0.8 we offer the possibility to use AKS or EKS as Kubernetes providers. That is introduced with modules mechanism, so we launched the first four modules:

File: docs/home/howto/MODULES.md [hunk 3]
```diff
@@ -2,99 +2,76 @@
...
 ## Introduction
 
-In version 0.8 of epiphany we are introducing modules. Modularisation of epiphany environment will result in: 
- * smaller code bases for separate areas, 
- * simpler and faster test process, 
- * interchangeability of elements providing similar functionality (eg.: different kubernetes providers), 
- * faster and more focused release cycle. 
- 
-Those and multiple other factors (eg.: readability, reliability) influence this direction of changes. 
+In version 0.8 of Epiphany we are introducing modules. Modularization of Epiphany environment will result in:
```
Comment (plirglo) [inline docs/home/howto/MODULES.md:5]: Maybe:
In version 0.8 of Epiphany we introduce modules

File: docs/home/howto/MODULES.md [hunk 4]
```diff
@@ -2,99 +2,76 @@
...
+* [AWS Basic Infrastructure](https://github.com/epiphany-platform/m-aws-basic-infrastructure) (AwsBI) module
+* [AWS EKS](https://github.com/epiphany-platform/m-aws-kubernetes-service) (AwsKS) module
 
-As it might be already visible there are 2 paths provided: 
- * Azure related, using AzBI and AzKS modules, 
- * AWS related, using AwsBI and AwsKS modules. 
- 
-Those "... Basic Infrastructure" modules are responsible to provide basic cloud resources (eg.: resource groups, virtual networks, subnets, virtual machines, network security rules, routing, ect.) which will be used by next modules. So in this case, those are "... KS modules" meant to provide managed kubernetes services. They use resources provided by basic infrastructure modules (eg.: subnets or resource groups) and instantiate managed kubernetes services provided by cloud providers. Last element in both those cloud provider related paths is Classic Epiphany installed on top of resources provided by those modules using `any` provider.    
+Those 4 modules with the Classic Epiphany used with `any` provider allow change of on-prem Kubernetes cluster with managed Kubernetes services.
```
Comment (sk4zuzu) [inline docs/home/howto/MODULES.md:39]:
```suggestion
Those 4 modules together with classic Epiphany used with `any` provider allow replacing of on-prem Kubernetes cluster with managed Kubernetes services.
```

File: docs/home/howto/MODULES.md [hunk 5]
```diff
@@ -2,99 +2,76 @@
...
+
+As it might be already visible there are 2 paths provided:
+
+* Azure related, using AzBI and AzKS modules,
+* AWS related, using AwsBI and AwsKS modules.
+
+Those "... Basic Infrastructure" modules are responsible to provide basic cloud resources (eg.: resource groups, virtual networks, subnets, virtual machines, network security rules, routing, ect.) which will be used by next modules. So in this case, those are "... KS modules" meant to provide managed Kubernetes services. They use resources provided by basic infrastructure modules (eg.: subnets or resource groups) and instantiate managed Kubernetes services provided by cloud providers. The last element in both those cloud provider related paths is Classic Epiphany installed on top of resources provided by those modules using `any` provider.
```
Comment (sk4zuzu) [inline docs/home/howto/MODULES.md:46]:
```suggestion
Those "... Basic Infrastructure" modules are responsible to provide basic cloud resources (eg.: resource groups, virtual networks, subnets, virtual machines, network security rules, routing, ect.) which will be used by next modules. So in this case, those are "... KS modules" meant to provide managed Kubernetes services. They use resources provided by basic infrastructure modules (eg.: subnets or resource groups) and instantiate managed Kubernetes services provided by cloud providers. The last element in both those cloud provider related paths is classic Epiphany installed on top of resources provided by those modules using `any` provider.
```

File: docs/home/howto/MODULES.md [hunk 6]
```diff
@@ -103,7 +80,7 @@ To prepare infrastructure we will use AzBI module to create resource group, vnet
   specification:
     name: your-cluster-name # <----- make unified with other places and build directory name
     admin_user:
-      name: operations # <----- don't change username
+      name: operations # <----- make sure os-user is correct
       key_path: /tmp/shared/vms_rsa # <----- use generated keyfile
     cloud:
       k8s_as_cloud_service: true # <----- make sure that flag is set, as it indicates usage of AKS service
```
Comment (sk4zuzu) [inline docs/home/howto/MODULES.md:86]:
```suggestion
      k8s_as_cloud_service: true # <----- make sure that flag is set, as it indicates usage of a managed Kubernetes service
```

File: docs/home/howto/MODULES.md [hunk 7]
```diff
@@ -103,7 +80,7 @@ To prepare infrastructure we will use AzBI module to create resource group, vnet
   specification:
     name: your-cluster-name # <----- make unified with other places and build directory name
     admin_user:
-      name: operations # <----- don't change username
+      name: operations # <----- make sure os-user is correct
       key_path: /tmp/shared/vms_rsa # <----- use generated keyfile
```
Comment (sk4zuzu) [inline docs/home/howto/MODULES.md:84]:
```suggestion
      key_path: /tmp/shared/vms_rsa # <----- use generated key file
```

**Review by sk4zuzu — CHANGES_REQUESTED** [review]

File: docs/home/howto/MODULES.md (hunk 3 above)
Comment (rafzei) [reply docs/home/howto/MODULES.md:5]: Or even 'introduced' :)

File: docs/home/howto/MODULES.md (hunk 2 above)
Comment (rafzei) [reply docs/home/howto/MODULES.md:32]: :+1:

**Review by atsikham — APPROVED** [review]
Looks better now 👍

**Review by sk4zuzu — APPROVED** [review]
LGTM :hugs:

**Pull request merged**